Makefile

```diff
@@ -9,10 +9,11 @@ PROJ = google_containers
 
 PKG_SOURCES := $(shell find pkg -name '*.go')
 
-node-problem-detector: ./bin/node-problem-detector
+dep:
+	which godep || go get github.com/tools/godep
 
-./bin/node-problem-detector: $(PKG_SOURCES) node_problem_detector.go
-	CGO_ENABLED=0 GOOS=linux godep go build -a -installsuffix cgo -ldflags '-w' -o node-problem-detector
+node-problem-detector: $(PKG_SOURCES) node_problem_detector.go dep
+	GOOS=linux godep go build -ldflags '-w -extldflags "-static"' -o node-problem-detector
```

Comment:
In fact currently, the journald integration won't work, because of the lack of systemd shared library in the container.

I'm fine with adding "-static" for now, but in #39 we don't need to do that because we'll be using fedora as the base image which should have the required libc libraries.